src/app/modules/main/wallet_section/transactions/io_interface.nim

```diff
@@ -43,6 +43,17 @@ method setIsNonArchivalNode*(self: AccessInterface, isNonArchivalNode: bool) {.b
 method estimateGas*(self: AccessInterface, from_addr: string, to: string, assetAddress: string, value: string, data: string): string {.base.} =
   raise newException(ValueError, "No implementation available")
 
+method transferEth*(self: AccessInterface, from_addr: string, to_addr: string, value: string,
+    gas: string, gasPrice: string, maxPriorityFeePerGas: string, maxFeePerGas: string,
+    password: string, uuid: string): bool {.base.} =
```

Comment:
same as before, I will stop there on the indentation ;) but the most of the function suffers from it

Reply:
I realize there is no real code style in Nim, but from python (which uses same syntax) this is what most of the autoformatter does (there is 4)